Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -35,6 +35,8 @@ + (UILocalNotification *)UILocalNotification:(id)json
   UILocalNotification *notification = [UILocalNotification new];
   notification.fireDate = [RCTConvert NSDate:details[@"fireDate"]] ?: [NSDate date];
   notification.alertBody = [RCTConvert NSString:details[@"alertBody"]];
+  notification.alertAction = [RCTConvert NSString:details[@"alertAction"]] ?: nil;
```

Comment:
Not a biggie, but I’m unsure what the point is of the extra ternary. Right now it reads to me like: if the result of converting is `nil` return `nil` instead.